DEVELOPMENT.md

```diff
@@ -0,0 +1,108 @@
+# Developing
+
+## Getting started
+
+Many of the instructions here replicate what's in the [tektoncd/pipeline development guide](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md), with a couple of caveats currently.
+
+1. This project currently does not support being built with `ko`
+2. This project has not yet been tested with GKE
+
+We would love to accomplish both of these tasks and to update this document, contributions welcome!
+
+### Checkout your fork
+
+The Go tools require that you clone the repository to the
+`src/github.com/tektoncd/dashboard` directory in your
+[`GOPATH`](https://github.com/golang/go/wiki/SettingGOPATH).
+
+To check out this repository:
+
+1. Create your own
+   [fork of this repo](https://help.github.com/articles/fork-a-repo/)
+1. Clone it to your machine:
+
+```shell
+mkdir -p ${GOPATH}/src/github.com/tektoncd
+cd ${GOPATH}/src/github.com/tektoncd
+git clone git@github.com:${YOUR_GITHUB_USERNAME}/dashboard.git
+cd dashboard
+git remote add upstream git@github.com:tektoncd/dashboard.git
+git remote set-url --push upstream no_push
+```
+
+_Adding the `upstream` remote sets you up nicely for regularly
+[syncing your fork](https://help.github.com/articles/syncing-a-fork/)._
+
+### Requirements
```

Comment:
Is it worth putting this "Requirements" section ahead of the checking out your fork as the checkout commands assume a GOPATH already exists?